Makefile

```
@@ -27,17 +27,20 @@ run-camera-simulator:
 	./camera-simulator/camera-simulator.sh
 
 run-telegraf:
-	cd telegraf && ./docker-run.sh
+	cd telegraf && ./run.sh
```

the file inside the telegraf folder is call docker-run.sh
```suggestion
	cd telegraf && ./docker-run.sh
```

---

i;ll change it to add that inside the makefile of telegraf and here just call that make target